pkgs/applications/virtualization/nvidia-docker/libnvc.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, lib, fetchFromGitHub, libelf, libcap, libseccomp }:
+
+with lib; let
+
+  modp-ver = "396.51";
```

Comment:
Actually, another thing regarding this: Doesn't this imply that this PR will only work for the 396.51 driver? Ideally we would somehow condition this on what the user uses in their NixOS system.

Reply:
No, thankfully not. This is just nixified from https://github.com/NVIDIA/libnvidia-container/blob/master/mk/nvidia-modprobe.mk#L9 basically. And for me at least that is not the version I'm running at (415.27) and 410 before. I'm assuming they only bump that when the API changes, not sure.
If I could use a fixed version, we wouldn't need anything in /run because it could just be referenced straight from the package. It would make everything much simpler, but then we need several versions of this package, and select those based on kernel/driver versions.